digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala

```diff
@@ -692,18 +692,23 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
       Nil
   }
 
-  def getViiteCurrentAndHistoryRoadLinksFromVVH(roadAddressesLinkIds: Set[Long], frozenTimeVVHAPIServiceEnabled : Boolean = false): (Seq[RoadLink], Seq[VVHHistoryRoadLink]) = {
+  def getViiteCurrentAndHistoryRoadLinksFromVVH(linkIds: Set[Long],
+                                                frozenVVHLinks: Boolean = false): (Seq[RoadLink], Seq[VVHHistoryRoadLink]) = {
     val fut = for{
-      f1Result <- vvhClient.historyData.fetchVVHRoadLinkByLinkIdsF(roadAddressesLinkIds)
-      f2Result <- if(frozenTimeVVHAPIServiceEnabled)vvhClient.frozenTimeRoadLinkData.fetchByLinkIdsF(roadAddressesLinkIds) else {vvhClient.roadLinkData.fetchByLinkIdsF(roadAddressesLinkIds)}
-      f3Result <- vvhClient.complementaryData.fetchByLinkIdsF(roadAddressesLinkIds)
+      f1Result <- vvhClient.historyData.fetchVVHRoadLinkByLinkIdsF(linkIds)
+      f2Result <- if(frozenVVHLinks)vvhClient.frozenTimeRoadLinkData.fetchByLinkIdsF(linkIds) else {vvhClient.roadLinkData.fetchByLinkIdsF(linkIds)}

```

Comment:
really think original naming was better. It was easier to see it was boolean value and not for example list

Reply:
Let's find a short name where type is clear. "useFrozenVVHLinks"?

Reply:
now that @oappicgi mentions that, it makes sense. "useFrozenVVHLinks" sounds good